### PR TITLE
test(custom profile): Add support in upgrade and longevity tests

### PR DESCRIPTION
**Pull request by yarongilor**
A new test with a basic flow, that could be used with
	custom-profile specific configuration and workload.
	It pre-loads custom dataset by initial load.
	The test runs additional custom profile load during the upgrade/rollback.
	It verifies the basic dataset after all nodes are upgraded.
Task: https://github.com/scylladb/qa-tasks/issues/21

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/upgrades/customer-profile/rolling-upgrade-custom-d1.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# workloads
+# Customer profiles directory path is: scylla-qa-internal/customer-profile/cust_d
+
+# To be used in keyspace_entire_test:
+cs_duration: '300m'  # TODO: measure new upgrade test duration and adjust this one.
+cs_user_profiles:
+    - scylla-qa-internal/custom_d1/write_stress_during_entire_test.yaml
+# write_stress_during_entire_test: cassandra-stress user no-warmup profile=/tmp/write_stress_during_entire_test.yaml  ops'(insert=1,read=2)' cl=ALL n=5000000 -mode cql3 native -rate threads=1000 -pop seq=1..5000000
+
+# To be used in keyspace_custom_preload:
```
Comment (fruch) [inline test-cases/upgrades/customer-profile/rolling-upgrade-custom-d1.yaml:12]: Please put it above the main load, it's a bit confusing like this

File: upgrade_test.py [hunk 2]
```diff
@@ -762,6 +762,107 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
 
         InfoEvent(message='all nodes were upgraded, and last workaround is verified.').publish()
 
+    def test_custom_profile_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
+        """
+        Run a load of a custom profile.
```
Comment (fruch) [inline upgrade_test.py:767]: Is this whole flow copied from a different test method ? Maybe we should consider sharing most of it ?

**Comment by yarongilor** [comment]
Started a basic test run of a 3 hours longevity in:
https://jenkins.scylladb.com/job/scylla-staging/job/yarongilor/job/longevity-custom-d1/4/

The test includes the following:
1. Fetching a json schema file for Gemini to use. It is taken from scylla-qa-internal private repository.
2. The schema is missing some part due to Gemini support issues ("map" columns, lists-of-UDTs).
3. The table configuration is fully applied.
4. The missing schema parts are: keyspace-RF, UDT definitions.
5. Gemini has also a limitation it doesn't support requests for specific queries.
6. The test uses GCE instances with NVMes. The hybrid RAID is still not implemented (in progress).

**Comment by fruch** [comment]
already introduce those in #6716

**Pull request closed without merging**